data.json

```diff
@@ -347,10 +347,19 @@
     "githubUsername": "rosajen27",
     "imageUrl": "https://raw.githubusercontent.com/rosajen27/rosajen27/master/rosajen27.png",
     "issueNumber": 165
-  },	{
+  },
+	{
+    "name": "Mohit Rakhade",
+    "githubUsername": "mohitrakhade20",
+    "imageUrl": "https://user-images.githubusercontent.com/48031017/98556319-eecabe00-22c8-11eb-8692-57d2efa3d3ab.png",
+    "issueNumber": 168
+  },
+  {
     "name": "Jayant Goel",
     "githubUsername": "JayantGoel001",
     "imageUrl": "https://user-images.githubusercontent.com/54479676/98487992-88a05580-224c-11eb-8e76-d8a59aa05bd4.png",
     "issueNumber": 169
+
```

Comment:
Not sure what the extra blank line is needed?

Reply:
please check now